### ISSUE TITLE
Storefront shoud use compact sizes for small viewports

### ISSUE THREAD

**Issue by mrosvik** [digdir/designsystemet#2105]
This should be done when the new typography variables are ready. The new typography variables comes with compact font-sizes for small viewports. 

We should use smaller headings and ingress on mobile: 
![Image](https://github.com/digdir/designsystemet/assets/1464915/5b01bddc-f2ee-4ae5-ba55-20f9eb4a6045)

Problem with long words: 
![Image](https://github.com/digdir/designsystemet/assets/1464915/9904f1b3-d9ee-408d-abef-ab58b6dd5353)

**Comment by Barsnes** [comment]
Blocked by #2103.

**Comment by mrosvik** [comment]
We should have a meeting to decide fontsizes for different breakpoints in Storefront.

**Comment by mrosvik** [comment]
> Blocked by [#2103](https://github.com/digdir/designsystemet/issues/2103).

Maybe we don´t have to wait for this? Could we solve it in the Storefront first?

**Comment by eirikbacker** [comment]
Depends on #2508

**Comment by Barsnes** [comment]
Is this issue still relevant?

**Comment by Barsnes** [comment]
Closing this as we have a nice sizing technique in DS, and the storefront currently looks good as is.
Feel free to open this again, or create a new issue if further changes needs to be done
